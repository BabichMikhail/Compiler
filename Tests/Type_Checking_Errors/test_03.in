var 
	a : integer = 10;
	b : double = 10 ;
begin
	if a = true then 
		b := a; 
end.