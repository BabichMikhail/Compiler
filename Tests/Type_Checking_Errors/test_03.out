Error: Incompatible Types. Expected "Integer" but "Boolean" found. Line 5 Column 12
